src/scanner.h

```diff
@@ -40,7 +40,8 @@ typedef struct ScannerCtx
 	ScannerType scantype;
 	ScanKey		scankey;
 	int			nkeys,
-				norderbys;
+				norderbys,
+				limit;
```

Comment:
Maybe document that a limit of 0 is ignored? -- not obvious. Btw are we sure we don't want to make -1 the "ignore flag". 0 Seems like a valid limit value.

Reply:
I prefer to keep 0 as ignore as I think it is nice to make a struct memset to 0 as default as possible. Otherwise you have to know to set to -1 if you don't want to use it. Agree about a comment documenting it.